README.md

```diff
@@ -1023,6 +1023,7 @@ Table of Contents
   * [Awardspace.com](https://www.awardspace.com) — Free web hosting + a free short domain, PHP, MySQL, App Installer, Email Sending & No Ads.
   * [Bubble](https://bubble.io/) — Visual programming to build web and mobile apps without code, free with Bubble branding.
   * [Deploy Now](https://deploynow.space) — Deploy smarter. Deploy faster. Deploy Now. - Deploy up to 3 web projects from your GitHub repository for free.
+  * [DigitalOcean](https://www.digitalocean.com/) - Build and deploy 3 static sites for free on the App Platform Starter tier. 
```

Comment:
Maybe link to https://www.digitalocean.com/pricing so all their free stuff is obvious